carapace-server/src/main/java/org/carapaceproxy/server/certificates/ACMEClient.java

```diff
@@ -178,21 +197,15 @@ public Status checkResponseForChallenge(Challenge challenge) throws AcmeExceptio
      * @throws AcmeException
      */
     public void orderCertificate(Order order, KeyPair domainKeyPair) throws IOException, AcmeException {
-        String domain = order.getIdentifiers().get(0).getDomain();
+        List<String> domains = order.getIdentifiers().stream().map(e -> e.getDomain()).collect(Collectors.toList());
 
         // Generate a CSR (Certificate signing request) for all of the domains, and sign it with the domain key pair.
         CSRBuilder csrb = new CSRBuilder();
-        csrb.addDomain(domain);
-        //csrb.setOrganization("The Example Organization")
+        csrb.addDomains(domains);
         csrb.sign(domainKeyPair);
 
-        // Write the CSR to a file, for later use (p.e. renewal of the certificate)                     
```

Comment:
remove all commented code an useless comments

Reply:
this still have not been addressed

Reply:
All useless code lines have been removed